components/tag-list/tag-list.js

```diff
@@ -249,11 +334,9 @@ class TagList extends LocalizeCoreElement(ArrowKeysMixin(LitElement)) {
 		if (!this._hasResized) return;
 
 		requestAnimationFrame(async() => {
-			this._items = this._getTagListItems();
+			this._items = await this._getTagListItems();
 			if (!this._items || this._items.length === 0) return;
 
-			await Promise.all(this._items.map(item => item.updateComplete));
```

Comment:
This is no longer needed after moving the updateComplete into `_getTagListItems`